### PR TITLE
Update the ODL dependency to ODL 7.7.1 and release version to 7.5

### PR DESCRIPTION
**Pull request by xuzhg**
2) Update the release version to 7.5.0

### Issues

*This pull request fixes issue https://github.com/OData/WebApi/issues/2274.*

### Description

1) Update the ODL dependency to 7.7.1
2) Update the Web API release version to 7.5

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: test/UnitTest/Microsoft.AspNetCore.OData.Test/Microsoft.AspNetCore.OData.Test.csproj [hunk 1]
```diff
@@ -33,13 +34,14 @@
...
-    <PackageReference Include="Microsoft.OData.Edm" Version="7.6.1" />
-    <PackageReference Include="Microsoft.Spatial" Version="7.6.1" />
+    <PackageReference Include="Microsoft.OData.Core" Version="7.7.1" />
+    <PackageReference Include="Microsoft.OData.Edm" Version="7.7.1" />
+    <PackageReference Include="Microsoft.Spatial" Version="7.7.1" />
     <PackageReference Include="Moq" Version="4.7.137" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="System.Text.Json" Version="4.7.1" />
```
Comment (gathogojr) [inline test/UnitTest/Microsoft.AspNetCore.OData.Test/Microsoft.AspNetCore.OData.Test.csproj:44]: @xuzhg Might [this issue](https://github.com/OData/odata.net/issues/1878) in odata.net have any bearing on the version of System.Text.Json here?

File: sln/WebApiOData.Performance.Official.sln [hunk 2]
```diff
@@ -19,12 +19,12 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.CodeAnalysis|Any CPU.ActiveCfg = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.CodeAnalysis|Any CPU.Build.0 = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.Debug|Any CPU.ActiveCfg = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.Debug|Any CPU.Build.0 = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.Release|Any CPU.Build.0 = Release|Any CPU
+		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.CodeAnalysis|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (gathogojr) [inline sln/WebApiOData.Performance.Official.sln:22]: @xuzhg I hope the configuration change here from Release to Debug was not inadvertent...

**Review by gathogojr — APPROVED** [review]
Left two minor comments otherwise LGTM

File: test/UnitTest/Microsoft.AspNetCore.OData.Test/Microsoft.AspNetCore.OData.Test.csproj (hunk 1 above)
Comment (xuzhg) [reply test/UnitTest/Microsoft.AspNetCore.OData.Test/Microsoft.AspNetCore.OData.Test.csproj:44]: When we fix [this issue](https://github.com/OData/odata.net/issues/1878), we won't this package reference at all. But, now, we have to include this package reference.

File: sln/WebApiOData.Performance.Official.sln (hunk 2 above)
Comment (xuzhg) [reply sln/WebApiOData.Performance.Official.sln:22]: That's VS changes, not from my change. However, i don't think it's a big deal, so, i keep the change in the PR.

**Pull request merged**